lk2nd/panel/oem_panel.c

```diff
@@ -43,6 +43,41 @@ uint32_t oem_panel_max_auto_detect_panels()
 #define _panel_select(panel)	panel_##panel##_select
 #define panel_select(panel)	_panel_select(panel)
 
+#if TARGET_MSM8916
+static int target_tps65132_ctrl(uint8_t enable)
+{
+	if (enable) {
+		/* for tps65132 ENP pin */
+		gpio_tlmm_config(enp_gpio.pin_id, 0,
+			enp_gpio.pin_direction, enp_gpio.pin_pull,
+			enp_gpio.pin_strength,
+			enp_gpio.pin_state);
+		gpio_set_dir(enp_gpio.pin_id, 2);
+
+		/* for tps65132 ENN pin*/
+		gpio_tlmm_config(enn_gpio.pin_id, 0,
+			enn_gpio.pin_direction, enn_gpio.pin_pull,
+			enn_gpio.pin_strength,
+			enn_gpio.pin_state);
+		gpio_set_dir(enn_gpio.pin_id, 2);
+	} else {
+		gpio_set_dir(enp_gpio.pin_id, 0); /* ENP */
+		gpio_set_dir(enn_gpio.pin_id, 0); /* ENN */
+	}
+	return 0;
+}
+
+int target_ldo_ctrl(uint8_t enable, struct msm_panel_info *pinfo)
+{
+	int ret;
+
+	if (panel_select(LK1ST_PANEL) == panel_tianma_nt35521_5p5_720p_video_select)
```

Comment:
Would it make any sense to prepare a switch...case here if we ever need to add more panel quirks? Don't think it's really important for now, but something to think about in the new branch

Reply:
Can't switch case function pointers :(